### PR TITLE
Enforce memory limits on misbehaving users

### PR DESCRIPTION
**Pull request by KellerFuchs**

### CODE REVIEW

File: systemd/system/user-4045.slice.d/lowmem.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[Slice]
+MemoryLimit=8M
```
Comment (daurnimator) [inline systemd/system/user-4045.slice.d/lowmem.conf]: On old systemd (i.e. what we have now) you also need to add `MemoryAccounting=true`

**Comment by KellerFuchs** [comment]
@daurnimator Can you sign off on this?

**Comment by daurnimator** [comment]
Do we still need this? Have those users given up yet?
I'd prefer that what are essentially shadow bans don't make it into shell-etc....

**Comment by KellerFuchs** [comment]
@daurnimator Well, you made those changes with @lrvick, and just never commited them.
If you would rather ban those users, that's what LDAP is for.

**Comment by daurnimator** [comment]
> @daurnimator Well, you made those changes with @lrvick, and just never commited them.

yes. we didn't to commit them, as we didn't want the user to (obviously) know what was happening

> If you would rather ban those users, that's what LDAP is for.

A ban would just make them create a new user account. We just wanted to significantly and covertly constrain their resources

**Comment by KellerFuchs** [comment]
> > @daurnimator Well, you made those changes with @lrvick, and just never commited them.
>
> yes. we didn't to commit them, as we didn't want the user to (obviously) know what was happening

Well, that also broke all the deploy automation...

**Comment by daurnimator** [comment]
Can we add something to e.g. `.gitignore` so it doesn't break stuff?

**Comment by KellerFuchs** [comment]
That might work

**Pull request closed without merging**